crates/bevy_ecs/src/schedule/set.rs

```diff
@@ -34,7 +39,37 @@ pub trait SystemSet: DynHash + Debug + Send + Sync + 'static {
     }
 
     /// Creates a boxed clone of the label corresponding to this system set.
-    fn dyn_clone(&self) -> Box<dyn SystemSet>;
+    fn dyn_leak(&self) -> &'static dyn SystemSet;
+    fn dyn_intern(&self) -> Interned<dyn SystemSet>
+    where
+        Self: Sized,
```

Comment:
Doesn't requiring `Self: Sized` contradict the `dyn_` part of this function's name?

Reply:
You are right, I have chosen this name because I didn't want an conflict between `Intern::intern` (which is implemented by `dyn SystemSet`) and this method. 

Since this was only a convenience method to be able to write `x.dyn_intern()` instead of `(&x as &dyn SystemSet).intern()`, I removed it.